content/docs/3_reference/7_plugins/1_extensions/0_panel-login/reference-extension.txt

```diff
@@ -1,11 +1,15 @@
 Title:  Panel login
 
 ----
+Text:
 
-Since: 3.2.0
+<since v="3.6.0">
+<warning>
+Panel views have been replaced with (link: docs/reference/plugins/extensions/panel-areas text: Panel areas) in 3.6.0.
+</warning>
+</since>
 
```

Comment:
We could use the `Deprecated`  field instead:

```
Deprecated:
3.6.0|Panel views have been replaced with (link: docs/reference/plugins/extensions/panel-areas text: Panel areas)

Reply:
With the  shift to separate docs for 3.6, these pages should be deleted, anyway, shouldn't they?

Reply:
True 👍